common/app/views/fragments/containers/facia_cards/auTerritorySelector.scala.html

```diff
@@ -0,0 +1,77 @@
+@import views.support.RenderClasses
+@(containerDefinition: layout.FaciaContainer)(implicit request: RequestHeader)
+
+<div class='fc-aus-territory__container'>
+    <button
+    class='fc-aus-territory__accordion-header'
+    data-link-name="AU-NSW-expand"
```

Comment:
I may be misunderstanding this (apologies if so) but should this data-link-name value be dynamic depending on the territory of the container? Presumably the 'NSW' bit could be replaced with  some mutation of `@containerDefinition.territoryName`?

Reply:
Good catch!